README.md

```diff
@@ -1,3 +1,127 @@
-# field_monitoring
+![](RackMultipart20220705-1-ruigrg_html_70e3b217e6f91606.png)
+
+# ATLAS Service Template Specification
+
+# Name: Crop Status Monitoring
+
+# Author(s): AgroApps (AGA)
+
+_The &quot;__Crop Status Monitoring&quot; ATLAS Service Template_
+
+This service will deliver a number of EO derived indicators for crop status monitoring consisting of:
+
+- Vegetation Indices
+
+NDVI
+
+- Biophysical Parameters for annual crops
+
+Biomass
+
+LAI
+
+- Chl Content
+- Canopy Water Content
+
+and a number of farm services, such as:
+
+- Yield Estimation
+- Crop Residues
+
+The system for the calculation and application of the VIs will be optimized with respect to crop types. To ensure high temporal coverage of the data, the system will utilize data from optical satellite sensors, namely the Sentinel-2.
+
+The provider of the Crop Status Monitoring Service is **AgroApps P.C. (AGA)**. The service is provided through a centralized integration RESTful API.
+
+## INTRODUCTION
+
+The **Crop Status Monitoring (CSM) Service**, is an earth observation crop monitoring module that exploits multispectral satellite data, and through the use of validated algorithms, provides crop monitoring and farming services products to the external client. The aim of the **CSM service** is to provide farmers with information on their crops&#39; health status, through a biophysical proxy, namely the NDVI and biophysical parameters (Leaf Area Index – LAI, Chlorophyll content – Chl, above ground crop biomass production, NDWI – canopy water status). Another aim is to provide estimations of the expected yield and crop residues. The CSM service multi-device application component ( **web-based or mobile** ) can help farmers identify anomalies in plant performance during the growing season, by comparing consecutive satellite images of high resolution (10 m or 20 m) depicting the status of their parcels.
```

Comment:
Too implementation specific when mentioning satellite. Other potential implementors will provide some of the indices by driving robots through field, others by drone fly-overs.